packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -361,6 +361,9 @@ export class Loan {
    * @param {string} metadata.vaultId Vault id
    * @param {string| string[]} metadata.amounts In "amount@symbol" format
    * @param {string} metadata.from Address from transfer tokens
+   * @param {TokenPaybackAmount[]} metadata.loans
```

Comment:
```suggestion
   * @param {TokenPaybackAmount[]} [metadata.loans]
```
optional

Reply:
Square brackets are not used for all the other fields, therefore I think it will look better without []

Reply:
`[]` indicates the optionality. so optional fields should have it.

Reply:
Actually it will not be optional, if the `PaybackLoanMetadataV2` is used, Now metadata will be one of `PaybackLoanMetadata` or `PaybackLoanMetadataV2` types

Reply:
oh you changed it. should be ok now.